client/application/ArtLocationInteractor.js

```diff
@@ -0,0 +1,84 @@
+// //Interactor goes here
```

Comment:
This should eventually be called something like `CreateNewArtLocationInteractor`. Naming of interactors is something we should discuss. I think they need to be verb phrases. But we should talk about different naming at different layers. For example, at first glance persistence gateway operations are CRUD operations. So should they be named as such? I was thinking interactors are named from a user perspective. They don't think of CRUD per se. But they think of similar things. Think of the File menu: New, Open, Save, Duplicate, etc. So does `NewLocationInteractor` make sense? That sounds like a noun because "Make" or "Create" is implied.  So that brings us to `CreateNewArtLocationInteractor`. Seems kind of verbose. Anyway...that's why we should chat about this.

Reply:
Sounds good!